src/glue/mod.rs

```diff
@@ -1,7 +1,10 @@
 #![cfg(feature = "sled-storage")]
 use {
-    crate::{execute, storages::SledStorage, ExecuteError, Payload, Query, Result},
+    crate::{execute, storages::SledStorage, ExecuteError, Payload, Query, Result, Value},
     futures::executor::block_on,
+    sqlparser::ast::{
+        Expr, Ident, ObjectName, Query as AstQuery, SetExpr, Statement, Value as Literal, Values,
```

Comment:
Ok, lets use `Value as Literal` to `Value as AstValue` like `Query as AstValue` because we now have `data::Literal`.

Reply:
Oopsie, good point.
Thanks.